evm.md

```diff
@@ -1280,19 +1280,21 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
          <account>
            <acctID> ACCT </acctID>
            <balance> BAL </balance>
+           <nonce> NONCE </nonce>
            ...
          </account>
          <callGas> GCALL </callGas>
-      requires VALUE >Int BAL orBool CD >=Int 1024
+      requires VALUE >Int BAL orBool CD >=Int 1024 orBool notBool #rangeNonce(NONCE)
```

Comment:
Here I could create a new exceptional status code  like `EVMC_NONCE_EXCEEDED` for the case in which `#rangeNonce` is `False`

Reply:
Does Geth make a new code like this? If so, we should as well.

It goes in `network.md` to do so.

Also, you may have to change some other parts of the semantics too. Do `git grep EVMC_SUCCESS` to see all the places that might need to be changed.